k8s/namespaces/ccd/ccd-case-print-service/ccd-case-print-service.yaml

```
@@ -21,7 +13,3 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/case-print-service:prod-f7d2085-20210807004743
```

https://github.com/hmcts/cnp-flux-config/blob/master/apps/ccd/ccd-case-print-service/image-policy.yaml
```suggestion
      image: hmctspublic.azurecr.io/ccd/case-print-service:prod-f7d2085-20210807004743  #{"$imagepolicy": "flux-system:ccd-case-print-service"}
```